airgun/views/virtwho_configure.py

```diff
@@ -19,13 +21,52 @@
 )
 
 
+class StatusIcon(GenericLocatorWidget):
+    """The status for virtwho configure can be: green, gray.
+
+    green: The virt-who report has not arrived within the interval,
+         which indicates there was no change on hypervisor
+    gray: The configuration was not deployed yet or the virt-who was
+        unable to report the status
+    """
+
+    def __init__(self, parent, locator=None, logger=None):
+        """Provide default locator value if it wasn't passed"""
+        if not locator:
+            locator = ".//span[contains(@class, 'virt-who-config-report-status')]"
+        Widget.__init__(self, parent, logger=logger)
+        self.locator = locator
+
+    @property
+    def color(self):
+        """Returns string representing icon color: 'gray', 'green' or'unknown'.
+        rgba is created for chrome.
+        rbg is created for firefox.
+        """
+        colors = {
+            'rgba(54, 54, 54, 1)': 'gray',
+            'rgba(63, 156, 53, 1)': 'green',
+            'rgb(63, 156, 53)': 'green',
+            'rgb(54, 54, 54)': 'gray',
+        }
+        return colors.get(
+            self.browser.element(self, parent=self.parent).value_of_css_property('color'),
+            'unknown'
+        )
+
+    def read(self):
+        """Returns current icon color"""
+        return self.color
+
```

Comment:
According to [widgetastic guidelines](https://widgetasticcore.readthedocs.io/en/latest/guidelines.html), all widgets should provide both `read()` and `fill()` methods. If `fill()` doesn't make sense for widget (as in your case), it should raise `ReadOnlyWidgetError` exception (see e.g. [`ItemsListReadOnly` in `airgun/widgets.py`](https://github.com/SatelliteQE/airgun/blob/master/airgun/widgets.py#L430)).